mobile/lib/services/album.service.dart

```diff
@@ -441,4 +444,33 @@ class AlbumService {
       return false;
     }
   }
+
+  Future<Album?> getAlbumByName(String name, bool remoteOnly) async {
+    return _db.albums
+        .filter()
+        .optional(remoteOnly, (q) => q.localIdIsNull())
+        .nameEqualTo(name)
+        .sharedEqualTo(false)
```

Comment:
what if the user shared their automatically-created album (e.g. camera)?

Reply:
The PR's primary goal is syncing the information from the local database to the remote database. For example, if you choose the auto-created `Camera` album. This function will search if there is such album's name existed on the Immich server, if not, then it will create a remote album with the name `Camera` and start adding the uploaded assets to it.